### ISSUE TITLE
Add a page-specific <h1> to all pages

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#834]
Part of #820

**Repositories list**
https://arclight-demo.projectblacklight.org/repositories

- [x]  Add `<h1 class="sr-only">Repositories</h1>`  immediately inside the `main-container` div, immediately above `<div class="al-search-breadcrumb">`

**Individual repository**
https://arclight-demo.projectblacklight.org/repositories/duke-rubenstein

Option 1:  Add h1, demote redundant h2 = consistent structure
- [ ]  Add `<h1 class="sr-only">[repository name]</h1>`  immediately inside the `main-container` div, immediately above `<div class="al-search-breadcrumb">`
- [ ] In `al-repository-information`, change existing `<h2>[repository name]</h2>`
- [ ] Apply `.al-repository .al-repository-information h2` css to the resulting `<span>` 

OR Option 2: Convert existing h2 to h1 = possibly easier?
- [x] In `al-repository-information`, change existing `<h2>[repository name]</h2>` to `<h1>[repository name]</h1>`
- [x] Apply `.al-repository .al-repository-information h2` css to the resulting `<h1>`  

**Search results**
https://arclight-demo.projectblacklight.org/?utf8=%E2%9C%93&search_field=all_fields&q=papers

- [x] Blacklight has `<h1 class="sr-only top-content-title">Search</h1>`, but we're not getting it in ArcLight. Either use the Blacklight version or create an equivalent? 

## Special search results pages
(These are technically search results; but we are presenting them as a specific browse-like destinations, and I think the h1 should reflect their difference from a generic search result.)

**Repository search results ("View all of our collections")**
https://arclight-demo.projectblacklight.org/catalog?f%5Blevel_sim%5D%5B%5D=Collection&f%5Brepository_sim%5D%5B%5D=David+M.+Rubenstein+Rare+Book+and+Manuscript+Library

- [x]  Replace "Search" with "Collections : [repository name]" in the `<h1>`

**Collections page**
https://arclight-demo.projectblacklight.org/collections

- [x]  Replace "Search" with "Collections" in the `<h1>`

## No change required
**Collection/component show page**
- `<h1>` already present
